### PR TITLE
Add className prop to series

### PR DESCRIPTION
**Pull request by mcnuttandrew**
#101

### CODE REVIEW

File: src/lib/plot/series/mark-series.js [hunk 1]
```diff
@@ -49,7 +51,7 @@ class MarkSeries extends AbstractSeries {
     const yFunctor = this._getAttributeFunctor('y');
 
     return (
-      <g className="rv-xy-plot__series rv-xy-plot__series--mark"
+      <g className={`${predefinedClassName} ${className || ''}`}
```
Comment (superbeefy) [inline src/lib/plot/series/mark-series.js]: just set a default prop on className so you don't have to do this pipe syntax

File: src/lib/plot/series/line-series.js [hunk 2]
```diff
@@ -70,7 +72,7 @@ class LineSeries extends AbstractSeries {
     return (
       <path
         d={d}
-        className="rv-xy-plot__series rv-xy-plot__series--line"
+        className={`${predefinedClassName} ${className || ''}`}
```
Comment (superbeefy) [inline src/lib/plot/series/line-series.js]: default props on className

File: src/lib/plot/series/heatmap-series.js [hunk 3]
```diff
@@ -53,7 +55,7 @@ class HeatmapSeries extends AbstractSeries {
     const yDistance = this._getScaleDistance('y');
     return (
       <g
-        className="rv-xy-plot__series rv-xy-plot__series--heatmap"
+        className={`${predefinedClassName} ${className || ''}`}
```
Comment (superbeefy) [inline src/lib/plot/series/heatmap-series.js]: defaultProp on className

File: src/lib/plot/series/bar-series.js [hunk 4]
```diff
@@ -87,8 +90,7 @@ class BarSeries extends AbstractSeries {
     const itemSize = (distance / 2) * 0.85;
 
     return (
-      <g
-        className="rv-xy-plot__series rv-xy-plot__series--bar"
+      <g className={`${predefinedClassName} ${className || ''}`}
```
Comment (superbeefy) [inline src/lib/plot/series/bar-series.js]: default prop on className

File: src/lib/plot/series/area-series.js [hunk 5]
```diff
@@ -58,7 +58,7 @@ class AreaSeries extends AbstractSeries {
     return (
       <path
         d={d}
-        className="rv-xy-plot__series rv-xy-plot__series--line"
+        className={`${predefinedClassName} ${className || ''}`}
```
Comment (superbeefy) [inline src/lib/plot/series/area-series.js]: default prop on className

**Comment by chrisirhc** [comment]
Alternatively, you can use the classnames package, so you can avoid doing any string concat operation.

**Pull request merged**